host-configs/llnl/blueos_3_ppc64le_ib_p9/clang@upstream_nvcc_xlf.cmake

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+# other BLT Project Developers. See the top-level COPYRIGHT file for details
+# 
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+#------------------------------------------------------------------------------
+# Example host-config file for the blue_os cluster at LLNL
+#------------------------------------------------------------------------------
+#
+# This file provides CMake with paths / details for:
+#  C/C++:   Clang
+#  Fortran: XLF
+#  MPI
+#  Cuda
+# 
+#------------------------------------------------------------------------------
```

Comment:
New blt host-config for blueos with Clang+XLF+Cuda.

Adapted from Axom: https://github.com/LLNL/axom/blob/develop/host-configs/rzansel-blueos_3_ppc64le_ib_p9-clang%40upstream_nvcc_xlf.cmake